dockers/skylab/HCA_post_processing/optimus_HCA_loom_merge.py

```diff
@@ -58,11 +61,10 @@ def main():
     input_id_list = []
     input_name_list = []
 
-    with loompy.new(args.output_loom_file) as dsout:
+    with loompy.new(output_loom_file) as dsout:
         for i in range(len(loom_file_list)):
             loom_file = loom_file_list[i]
             with loompy.connect(loom_file) as ds:
-                ds.ca['cell_names'] = ds.ca['cell_names'] + "-" + str(i)
```

Comment:
Why is this being removed?

Reply:
It is moved to the merged file since we don't need to change the original loom files, only the merged file should have -0 -1 and so on